src/main/resources/public/js/app.js

```diff
@@ -1,16 +1,20 @@
 var gameModel;
 
-//This function will be called once the page is loaded.  It will get a new game model from the back end, and display it.
 $( document ).ready(function() {
-
+  // Handler for .ready() called.
   $.getJSON("model", function( json ) {
-    displayGameState(currModel);
-    gameModel = json;
+  gameModel = json;
+    console.log( "JSON Data: " + json );
```

Comment:
I am almost certain that you shouldn't have everything in the game model, because if someone so much as glances as the console, there goes the whole game.